docs/source/documentation.rst

```diff
@@ -365,6 +365,49 @@ This will generate **'axon_morphometrics.xlsx'** file in each of folders::
     ---- pixel_size_in_micrometer.txt
     ---- axon_morphometrics.xlsx
 
+Morphometrics file
+~~~~~~~~~~~~~~~~~~
+
+The resulting **'axon_morphometrics'** file will contain the following columns headings:
+
+x0
+                    Axon X centroid position in pixels. Extracted with `skimage.measure.regionprops <scikit-image https://scikit-image.org/docs/stable/api/skimage.measure.html#regionprops>`_.
+
+y0
+                    Axon Y centroid position in pixels. Extracted with `skimage.measure.regionprops <scikit-image https://scikit-image.org/docs/stable/api/skimage.measure.html#regionprops>`_.
+
+gratio
+                    Ratio between the axon equivalent diameter and the axon+myelin (fiber) equivalent diameter.
```

Comment:
Perhaps we can skip adding the formula, but it would be very important to mention that gratio is computed using the area of the axon and myelin objects. `gratio = axon area / (axon area + myelin area)`

Reply:
I agree, I think the formula is too much but I will add a mention that the "equivalent diameter" is based on the area.

Also, the g-ratio is the inner/outer diameter (`gratio = axon_diameter/aonmyelin_diameter`), so the formula with the area would have a sqrt, no? `gratio = sqrt(axon area / (axon area + myelin area))`.

Reply:
yes it would be great to add a formula! @mathieuboudreau can confirm what is the correct equation

Reply:
> Also, the g-ratio is the inner/outer diameter (`gratio = axon_diameter/aonmyelin_diameter`), so the formula with the area would have a sqrt, no? `gratio = sqrt(axon area / (axon area + myelin area))`.

Yes, you are correct, it is `gratio = sqrt(axon area / (axon area + myelin area))`

Reply:
Should I go with the general definition `gratio = axon_diameter/axonmyelin_diameter`, or with what ADS actually do `np.sqrt(axon_area / axonmyelin_area)` ?

Reply:
Although it doesn't matter whichever formula we use as the result is the same, I'm favor of the actual formula we do in ADS, that is, `np.sqrt(axon_area / axonmyelin_area)`. @mathieuboudreau and @jcohenadad can chime in as well.

Reply:
@jcohenadad, @vs74
We need an agreement on the g-ratio definition. I wrote "in words" the general definition based on the ratio of **diameters**. But we want to add a formula, and the ADS formula is the square root of the **areas** ratio.  I think both the definition and formula should refer to the same metric for clarity, either diameter or area.

Currently in ADS, both definitions gives the same results, because the diameter is computed with the area of a circle.
However, at some point, we will add the option to compute the diameter as the minor axis of an ellipse as well. In that case, the current g-ratio with areas in ADS will have to change for the definition based on diameters. This is why I prefer the diameter definition.

For this PR, I see 2 options, which one should we use?

**Option 1, diameter:**
Ratio between the axon equivalent diameter and the axon+myelin (fiber) equivalent diameter.
`gratio = axon_diameter/axonmyelin_diameter`

**Option 2, area:**
Square root of the ratio between the axon area and the axon+myelin (fiber) area.
`gratio = sqrt(axon_area / axonmyelin_area)`

Reply:
**I am firmly voting for Option 1.** It's important that we use the same definition (equation) as the literature (which is that gratio = axon_diameter/axonmyelin_diameter), but then as you've done we should specify that because this is only valid for circles (ideal case), we're calculating the diameters from equivalent circles of the same are.

Reply:
The way how current g-ratio would definitely change with the inclusion of minor axis of an ellipse to compute diameter and we would change the g-ratio to generalized definition(**option 1**) rather than what is implemented in ADS now ( using **option 2**)

Hence, I am also in favor of **option 1**, a generalized definition.

Reply:
+1 for option 1

Reply:
Thank you for your feedback!
Fixed in 055c9d6.